content/desktop/release-notes.md

```diff
@@ -24,11 +24,71 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+# 4.24.0
+
+{{< release-date date="2023-09-28" >}}
+
+{{< desktop-install all=true version="4.24.0" build_path="/" >}}
+
+### Upgrades
+
+- [Compose v2.22.0](https://github.com/docker/compose/releases/tag/v2.22.0)
+- [Go 1.21.1](https://github.com/golang/go/releases/tag/go1.21.1)
+- [Wasm](../desktop/wasm/_index.md) runtimes:
+  - wasmtime, wasmedge `v0.2.0`.
+  - lunatic, slight, spin, and wws`v0.9.1`.
+  - Added wasmer wasm shims.
+
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Docker Init:
+  - Fixed an issue formatting Dockerfile file paths for ASP.NET projects on Windows.
+  - Improved performance on language detection for large directories with lots of files.
+- Added a timeout to polling for resource usage stats used by the Containers dashboard. Fixes [docker/for-mac#6962](https://github.com/docker/for-mac/issues/6962).
+- containerd integration:
+  - Implement push/pull/save image events
+  - Implement pulling legacy schema1 images
+  - Implement `docker push --all-tags`
+  - Implement counting containers using a specific image (visible for example in `docker system df -v`)
+  - Validate pulled image names are not reserved
+  - Handle `userns-remap` daemon setting
+  - Fix legacy builder build errors when multiple COPY/ADD instructions are used
+  - Fix `docker load` causing pool corruption which could some subsequent image related operations
+  - Fix not being able to reference images via truncated digest with a `sha256:` prefix
+  - Fix `docker images` (without `--all`) showing intermediate layers (created by the legacy classic builder)
+  - Fix `docker diff` containing extra differences
+  - Change `docker pull` output to match the output with containerd integration disabled
+-  You can now view and manage the Docker Engine state from the dashboard.
+- Fixed a grammatical error in Kubernetes status message. See [docker/for-mac#6971](https://github.com/docker/for-mac/issues/6971).
+- Docker containers use all host CPU cores by default.
+- Improved inter-process security in dashboard UI.
+
+#### For Mac
+
+- Fix a kernel panic on Apple silicon Macs with macOS version below 12.5. See [docker/for-mac#6975](https://github.com/docker/for-mac/issues/6975)
```

Comment:
```suggestion
- Fix a kernel panic on Apple silicon Macs with macOS version below 12.5. See [docker/for-mac#6975](https://github.com/docker/for-mac/issues/6975).
```

Reply:
Or maybe ⬇️ to have the same format as the next sentences?
```suggestion
- Fixed a kernel panic on Apple silicon Macs with macOS version below 12.5. Fixes [docker/for-mac#6975](https://github.com/docker/for-mac/issues/6975).
```

Reply:
I didn't put fixes, as it redirects to the comment here. Not sure if the whole issue is supposed to be fixed, or only that specific user. 🤔 💭